www/doc/2.1/tutorial.html

```diff
@@ -219,11 +233,12 @@ <h2>Herbie's results</h2>
     <img width="100%" src="problematic-xim.png" />
   </figure>
 
-  <p>The drop in accuracy once <code>xim</code> is bigger than
-  about <code>1e150</code> really stands out, but you can also see
-  that Herbie's alternative more accurate for smaller <code>xim</code>
-  values, too. You can also change the graph to plot accuracy
-  versus <code>xre</code> instead:</p>
+<p>The drop in accuracy (due to <a href="https://en.wikipedia.org/wiki/Floating-point_arithmetic#Dealing_with_exceptional_cases">floating-point overflow</a>)
```

Comment:
I think the parenthetical makes this whole sentence harder to parse. Let's say something more like,

> There's a really obvious drop in accuracy once `xim` gets bigger than about `1e150` (due to floating-point overflow), but you can also see ...